MotionMark/tests/simple/resources/simple-canvas-paths.js

```diff
@@ -25,8 +25,9 @@
 
 // === PAINT OBJECTS ===
 
-CanvasLineSegment = Utilities.createClass(
-    function(stage) {
+class CanvasLineSegment {
```

Comment:
Why is this class repeated here and in `core/resources/canvas-tests.js`? It looks like we either need to cleanup this or delete this whole directory.